test/functional/test-log.js

```diff
@@ -382,43 +381,18 @@ describe('Logging', () => {
       expect(error.expected).to.be.true;
     });
 
-    it('should create suffixed errors from message', () => {
+    it('should create user errors from message', () => {
       const error = log.createError('test');
       expect(error.expected).to.be.undefined;
       expect(error).to.be.instanceof(Error);
-      expect(isUserErrorMessage(error.message)).to.be.true;
-      expect(error.message).to.equal('test' + USER_ERROR_SENTINEL);
+      expect(isUserError(error)).to.be.true;
     });
 
-    it('should create suffixed errors from error', () => {
-      const error = log.createError(new Error('test'));
-      expect(error).to.be.instanceof(Error);
-      expect(isUserErrorMessage(error.message)).to.be.true;
-      expect(error.message).to.equal('test' + USER_ERROR_SENTINEL);
-    });
-
-    it('should only add suffix once', () => {
-      const error = log.createError(new Error('test' + USER_ERROR_SENTINEL));
-      expect(error).to.be.instanceof(Error);
-      expect(isUserErrorMessage(error.message)).to.be.true;
-      expect(error.message).to.equal('test' + USER_ERROR_SENTINEL);
-    });
-
-    it('should strip suffix if not available', () => {
-      const error = log.createError(new Error('test'));
-      expect(isUserErrorMessage(error.message)).to.be.true;
-
-      const noSuffixLog = new Log(win, RETURNS_FINE);
-      noSuffixLog.createError(error);
-      expect(isUserErrorMessage(error.message)).to.be.false;
-    });
-
-    it('should create other-suffixed errors', () => {
-      log = new Log(win, RETURNS_FINE, '-other');
```

Comment:
I'm not sure this would work, but please research, my info can be dated. We couldn't implement a boolean flag before and had to resort to concatenating an invisible suffix - obviously a very ugly solution. This is because in some circumstances the `Error` object was lost and only message could survive. For instance, in some cases the `error` argument would be missing in https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onerror on some browsers/use cases. 

Again, this might be a bit dated or we may no longer be interested in those use cases - please check.

/cc @cramforce